CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 1.6.2
+* Fix - Resolves fatal error when the plugin can't get the current user [#365](https://github.com/ebanx/woocommerce-gateway-ebanx/pull/365)
```

Comment:
This fix will only work if the error is really the current user, otherwise it's not a fix. So i think it's ok to say this fix works when the plugin can't get the current user.